src/io/flutter/FlutterMessages.java

```diff
@@ -29,6 +29,16 @@ public static void showError(String title, String message) {
                        NotificationType.ERROR));
   }
 
+  public static void showMessage(String title, String message) {
```

Comment:
W/ the addition of the `level` field, these method names now read as: `showError`, `showWarning`, and `showInfo`.